package/MDAnalysis/lib/transformations.py

```diff
@@ -205,6 +206,8 @@ def translation_matrix(direction):
 def translation_from_matrix(matrix):
     """Return translation vector from translation matrix.
 
+    >>> from MDAnalysis.lib.transformations import translation_matrix, \
```

Comment:
Here and everywhere else, please do stick to implicit line continuation (i.e. use brackets instead of `\`).